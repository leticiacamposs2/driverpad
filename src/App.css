@import '~antd/dist/antd.css';
@import '../node_modules/font-awesome/css/font-awesome.min.css'; 
