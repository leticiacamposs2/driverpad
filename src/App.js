import React from 'react';
import Routes  from './routes';
import { Layout } from 'antd';
import Banner from './components/Banner';
import Footer from './components/Footer';
import Header from './components/Header';
import logo from './assets/imgs/logo.svg';
import logoname from './assets/imgs/logoName.svg';
import logotipo from './assets/imgs/logo.png';
import './App.css';

const { Content } = Layout;

const App = () => (
  <>
    <Header logo={logo} alt="logo DriverPad" logoname={logoname} />
    <Content style={{ margin: '24px 16px 0' }}>
      <Banner logo={logotipo} alt="banner com o logotipo da driverpad"/>
      <Routes/>
    </Content>
    <Footer author="Leticia Campos" year="2020" />
  </>
);

export default App;
