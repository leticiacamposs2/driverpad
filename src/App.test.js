import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App', () => {
  it('deve renderizar corretamente em modo "debug"', () => {
    const component = shallow(<App debug />);

    expect(component).toMatchSnapshot();
  });

  it('deve renderizar corretamente sem nenhum props', () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
  });
});