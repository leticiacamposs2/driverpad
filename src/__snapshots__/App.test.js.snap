// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App deve renderizar corretamente em modo "debug" 1`] = `ShallowWrapper {}`;

exports[`App deve renderizar corretamente sem nenhum props 1`] = `ShallowWrapper {}`;
